### DISCUSSION TITLE
delete a whole line

### DISCUSSION THREAD

**Discussion by ousia** [orbitalquark/textadept#210] [Q&A] [answered]
I’m afraid I’m not able to invoke two commands to remove a whole line:

``` lua
keys['ctrl+k'] = function() buffer.del_line_right buffer.del_line_left end
```

Which is the right way to do this?

Many thanks for your help.

**Comment by rgieseke** [comment, marked as answer]
You need to call/execute the functions inside your `function()` by adding `()` to them, e.g. something like

```lua
keys['ctrl+k'] = function() 
  buffer.del_line_right()
  buffer.del_line_left()
end
```

**Reply by ousia** [reply]
Ok, this is what I was totally missing.

Many thanks for your help, @rgieseke.
